resources/js/features/messages/components/CreateMessageReplyForm/CreateMessageReplyForm.tsx

```diff
@@ -68,7 +71,9 @@ export const CreateMessageReplyForm: FC<CreateMessageReplyFormProps> = ({ onPrev
               </BaseButton>
 
               <BaseButton type="submit" disabled={!form.formState.isValid || mutation.isPending}>
-                {t('Submit')}
+                {auth!.user.displayName === senderUserDisplayName
+                  ? t('Submit')
+                  : t('Submit (as {{username}})', { username: senderUserDisplayName })}
```

Comment:
Would it be worth tracking which user actually sent the message? (add a `sent_by_id` column to the `messages` table)

Reply:
Went ahead and added this field in latest.